spec/acceptance/sensu_rbac_spec.rb

```diff
@@ -1,11 +1,17 @@
 require 'spec_helper_acceptance'
```

Comment:
Could you please add a comment block here that explains why we have sensu_rbac_spec.rb?